habitat_sim/__init__.py

```diff
@@ -9,7 +9,45 @@
 __version__ = "0.1.2"
 
 if not getattr(builtins, "__HSIM_SETUP__", False):
+    # TODO: all those import * should get removed, kept only for compatibility
+    #   with existing code
     from .nav import *
     from .agent import *
     from .simulator import *
-    from .bindings import *
+
+    # Bindings has a geo module, but we have our own. So take everything except
+    # that.
+    from .bindings import (
+        SceneNodeType,
+        GreedyFollowerCodes,
+        GreedyGeodesicFollowerImpl,
+        MultiGoalShortestPath,
+        PathFinder,
+        PinholeCamera,
+        SceneGraph,
+        SceneNode,
+        Sensor,
+        SensorSpec,
+        SensorType,
+        ShortestPath,
+        SimulatorConfiguration,
+    )
+
+    from . import agent, geo, gfx, logging, nav, scene, sensor, simulator, utils
```

Comment:
```suggestion
    import agent, geo, gfx, logging, nav, scene, sensor, simulator, utils
```

Reply:
As I said on Slack, this didn't really work :)